File: 2st/Riyeon/주사위 게임 2.java
class Solution {
    public int solution(int a, int b, int c) {
        if(a==b){
            if(b==c)
                return (a+b+c)*(a*a+b*b+c*c)*(a*a*a+b*b*b+c*c*c);
            else
                return (a+b+c)*(a*a+b*b+c*c);
        }else if(a==c || b==c)
            return (a+b+c)*(a*a+b*b+c*c);
        else
            return a+b+c;
        
    }
}

File: 2st/Yeonwoo/주사위 게임2.py
def solution(a, b, c):
    answer = 0
    if a!=b and b!=c and c!=a:
        answer = a+b+c
    elif a==b and b==c:
        answer = (a+b+c)*((a**2) + (b**2) + (c**2))*((a**3) + (b**3) + (c**3))
    else:
        answer = (a+b+c)*((a**2) + (b**2) + (c**2))
    return answer